code/style.css

```diff
@@ -1,3 +1,189 @@
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  background: #e0fbfc;
+  font-family: "Sora", sans-serif;
+  color: #293241;
+  margin: 0;
+}
+
+.githublogo {
+  width: 30px;
+  height: 30px;
+}
+
+h1 {
+  text-align: center;
+}
+
+.userinfo {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.userimg img {
+  border-radius: 50%;
+  width: 300px;
+  height: 300px;
+  object-fit: cover;
+}
+
+.userlink {
+  text-decoration: none;
+  color: #293241;
+}
+
+.userlink:visited {
+  text-decoration: none;
+  color: #293241;
+}
+
+.userlink:hover {
+  text-decoration: underline;
+  color: #ee6c4d;
+}
+
+hr {
+  width: 200px;
+  height: 1px;
+  background-color: #3d5a80;
+  border-radius: 5px;
+  line-height: 2px;
+  opacity: 0.85;
+}
+
+.logowrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+h2 {
+  display: inline;
+  margin-left: 5px;
+}
+
+.projectscontainer {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
+
+.projectinfo {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  background-color: #88b7d3;
+  width: 90%;
+  height: 200px;
+  margin-bottom: 20px;
+  box-shadow: 0 4px 8px 0 rgb(0 0 0 / 20%), 0 6px 20px 0 rgb(0 0 0 / 19%);
+  border-radius: 10px;
+  opacity: 0.85;
+  transition: 0.5s;
+}
+
+.projectinfo:nth-child(6) {
+  margin-bottom: 40px;
+}
+
+.repolink {
+  text-decoration: none;
+  color: #293241;
+  font-weight: bold;
+}
+
+.repolink:visited {
+  text-decoration: none;
+  color: #293241;
+}
+
+.repolink:hover {
+  text-decoration: underline;
+  color: #ee6c4d;
+}
+
+.chart {
+  width: 80%;
+  margin-bottom: 40px;
+  margin: auto;
+}
+
+@media (min-width: 768px) {
+  .projectscontainer {
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    margin-top: 20px;
+  }
+
+  .projectinfo:nth-last-child(-n + 2) {
+    margin-bottom: 60px;
+  }
+
```

Comment:
this looks like a setting that would adjust dynamically according to how many projects there are. Maybe use a construct like that on row 89?

`.projectinfo:nth-last-child(-n + 2)`

Reply:
Yeah, you're totally right. Thanks for pointing that out.